charter.md

```diff
@@ -0,0 +1,425 @@
+
+-	(c) Existing open source projects should continue to run through their existing technical governance structure to maintain cohesion and velocity. Projects approved by the TOC for inclusion in the CNCF will be ‘lightly’ subject to the Technical Oversight Committee.
+
+-	(d) A standard protocol to achieve committer status shall be established across projects based on an individual’s level and duration of contribution. Maintainer status is achieved through contribution to a given project over time and validation by peer committers.
+
+-	(e) New open source projects initiated in CNCF shall complete a project proposal template adopted by the TOC and be approved by the TOC for inclusion in CNCF. The TOC members shall be afforded sufficient time to discuss and review new project proposals. New project proposals shall include details of the roles in the project, the governance proposed for the project and identify alignment with CNCF’s role and values.
+
+#### 10. Marketing Committee
+
+-	(a) Composition: the Marketing Committee will be open to all members to participate. A Chair of the Marketing Committee shall be elected to develop meeting agendas, moderate discussions and help the committee progress against its goals. The Marketing Committee shall seek consensus where possible. Any issues that cannot reach a Rough Consensus in the Marketing Committee shall be referred to the Governing Board.
+
+-	(b) Responsibilities: The Marketing Committee shall be responsible for designing, developing and executing marketing efforts on behalf of the Governing Board.
+
+-	(c) If the Marketing Committee becomes too large to operate effectively, the Marketing Committee may choose to elect Marketing Board and delegate decision authority to the Marketing Board.
+
+#### 11. IP Policy
+
+-	(a) Any project that is added to the CNCF must have ownership of its trademark and logo assets transferred to the Linux Foundation.
+-	(b) Each project shall determine whether it will require use of an approved CNCF CLA. For projects that select to use a CLA, all code contributors will undertake the obligations set forth in the Apache contributor license agreement(s), altered only as necessary to identify CNCF as the recipient of the contributions, and which shall be approved by the Governance Board. See CNCF Contributor License Agreements available at https://github.com/cncf/cla. The process for managing contributions in accordance with this policy shall be subject to Governance Board approval.
+-	(c) All new inbound code contributions to the CNCF shall be (i) accompanied by a Developer Certificate of Origin sign-off (http://developercertificate.org) and (ii) made under the Apache License, Version 2.0 (available at http://www.apache.org/licenses/LICENSE-2.0), such license to be in addition to, and shall not supersede, obligations undertaken under the contribution license agreement(s) provided for in (b) above.
```

Comment:
https://developercertificate.org/ and http://www.apache.org/licenses/LICENSE-2.0